magut_eye_on_base:
  rotation_samples_number: 2
  camera_base_link: camera_link
  camera_marker_link: open_tip
  robot_base_link: world
  robot_end_link: tool0
